app/webpacker/controllers/toggle_class_controller.js

```diff
@@ -0,0 +1,19 @@
+import { Controller } from "stimulus"
+
+export default class extends Controller {
```

Comment:
I think this one deserves a test, you can have a look at files here: `/Users/jibees/dev/openfoodnetwork/spec/javascripts/stimulus`

Reply:
Yup definitely. I was looking into it today but didn't get the time + I'm not super familiar with jest. I'll finish it tomorrow and push it. Thank You for the reviews :slightly_smiling_face:

Reply:
Yup writing the test was easy. I just got stuck passing the `data..param` attribute correctly. The array needed to be enclosed within quotes but it took me ages to figure that out! :weary: Anyway, have a look at the test and let me know if there's anything. Cheers!

Reply:
Nice, thanks!